### PR TITLE
Set version 0.1.13

### PR DESCRIPTION
**Pull request by javierg**
pushing the version for gem publishing

### CODE REVIEW

File: lib/biran/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Biran
-  VERSION = '0.1.12'
+  VERSION = '0.1.13'
```
Comment (brlanier) [inline lib/biran/version.rb:2]: Your missing the change in Gemfile.lock as well

**Review by brlanier — CHANGES_REQUESTED** [review]
Do you also want to grab the nokogiri gem update as part of this release?

**Comment by javierg** [comment]
>Do you also want to grab the nokogiri gem update as part of this release?

No, that is not merged yet
should we?

**Review by brlanier — APPROVED** [review]
I am good with this without the other fix due to the rails issue. We need to check that out more
:shipit:

**Pull request merged**